BookingsApi/BookingsApi.UnitTests/Controllers/FeatureFlagsControllerTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿using BookingsApi.Contract.Configuration;
+using BookingsApi.Controllers;
+using BookingsApi.Services;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+
+namespace BookingsApi.UnitTests.Controllers
+{
+    public class FeatureFlagsControllerTests
+    {
+        private Mock<IFeatureFlagsService> _featureFlagsService;
```

Comment:
[GENERAL-COMMENT] Consider consistent use of Terminology (FeatureFlag / FeatureToggle)